diagnostics/physics/mld_eval.py

```diff
@@ -31,16 +39,18 @@ def plot_mld_eval(pp_root):
     )
     argo_grid = dict(lat=argo.lat, lon=argo.lon, lat_b=np.arange(-90, 90.1, 1), lon_b=np.arange(-180, 180.1, 1))
     argo_mld_winter = argo.sel(month=slice(1, 3)).mean('month')
+    logger.info("ARGO_MLD_WINTER: %s",argo_mld_winter)
 
     mom_to_argo = xesmf.Regridder(
-        model_grid[['geolon', 'geolat', 'geolon_c', 'geolat_c']].rename({'geolon': 'lon', 'geolat': 'lat', 'geolon_c': 'lon_b', 'geolat_c': 'lat_b'}),
+        model_grid[ config['rename_map'].keys() ].rename( config['rename_map'] ),
         argo_grid,
         method='conservative_normed',
         unmapped_to_nan=True,
         reuse_weights=False
     )
     mom_rg = mom_to_argo(mom_mld_winter)
     delta = mom_rg - argo_mld_winter
+    logger.info("DELTA: %s",delta)
 
     deboyer = xarray.open_dataset('/net2/acr/deBoyer2022/mld_dr003_ref10m.nc')
```

Comment:
Should we consider moving the hard-coded file path to the YAML config? The same applies to the Argo file.